lib/ethif/ethif.ml

```diff
@@ -16,6 +16,10 @@
  *
  *)
 open Lwt.Infix
+open Result
+
+let src = Logs.Src.create "ethif" ~doc:"Mirage Ethernet"
```

Comment:
I'm not sure about this... shouldn't we create the log instance in `connect` (and include some name)?   this way, if we have multiple ethernet interfaces, we can distinguish the individual log messages